rqt_gui/setup.py

```diff
@@ -0,0 +1,44 @@
+from setuptools import setup
+
+package_name = 'rqt_gui'
+setup(
+    name=package_name,
+    version='0.5.0',
+    package_dir={'': 'src'},
+    packages=['rqt_gui'],
+    data_files=[
+        ('share/' + package_name + '/resource', [
+            'resource/rqt.graffle',
+            'resource/rqt.png',
+            'resource/rqt.svg',
+        ]),
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+        ('share/' + package_name, ['package.xml']),
+    ],
+    install_requires=['setuptools'],
+    zip_safe=True,
+    author='Dirk Thomas',
+    author_email='dthomas@osrfoundation.org',
+    maintainer='Dirk Thomas',
+    maintainer_email='dthomas@osrfoundation.org',
+    keywords=['ROS'],
+    classifiers=[
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: Apache Software License',
+        'Programming Language :: Python',
+        'Topic :: Software Development',
+    ],
+    description=(
+        'rqt_gui provides the main to start an instance of the ROS integrated graphical user ' +
+        'interface provided by qt_gui.'
+    ),
+    license='BSD',
+    tests_require=['pytest'],
+    entry_points={
+        'console_scripts': [
+            'rqt_gui = rqt_gui.main:main',
```

Comment:
We remove `rqt_gui` from the console scripts again (with the other change only `rqt` is an entry point and goes to `bin`). Optionally we can restore a Python script `rqt_gui` and simply copy it to `lib/rqt_gui` (even if that doesn't make it `ros2 run`-able on Windows).

Reply:
rqt is now a console script that is installed to bin/ and rqt_gui is a python script that gets copied over to lib/rqt_gui